### ISSUE TITLE
Issuing NFT v1.1 Moderated tests

### ISSUE THREAD

**Issue by KubaMikolajczyk** [Joystream/atlas#1835]
# Description
We want to make sure that the experience with new changes is flawless for the users. Each session will take a max. 15 min.

**Comment by KubaMikolajczyk** [comment]
# Overview:
After the last round of moderated tests, we arrived at 3 final improvements in Wizard flow:
1️⃣  Explainer field was changed for a live "your NFT preview" due to low discoverability and overall usefulness for the users
2️⃣  Fee that is shown in the bottom action bar need to be also presented in the final summary for ease of scanning
3️⃣  Second screen of the form where the user chooses listing type (open, English, fixed etc) need to be dropped - there is no need to introduce those abstract concept to users as those confused all testers - instead we will show all possible options in one form under "put on marketplace"

If you want to know more about previous test click here -> https://github.com/Joystream/atlas/issues/1694 

### Amount of participants: 2/6

# Questions
### Have you already uploaded any video to Joystream studio?
👤 Tester 1️⃣  ❌
👤 Tester 2️⃣  ✅
### Have you bought or issued any NFT?
👤 Tester 1️⃣  ✅ 
👤 Tester 2️⃣  ✅
### What marketplaces you already used?
👤 Tester 1️⃣  Opensea
👤 Tester 2️⃣  Opensea, Singular, Pankake swap, checks out new marketplaces
### Have you participated in previous Issuing NFT test?
👤 Tester 1️⃣  ✅
👤 Tester 2️⃣  ❌

# Task
"You are publishing a video to Joystream studio, all video assets and information have been prefilled for you. Now you want to sell your video as an NFT and you don’t want to get bids lower than 100 000 tJOYS"

#### Prototype link:
[https://www.figma.com/proto/NdNx5jV2R2IDeMflnWPMD7/Issuing-NFT?page-id=1597%3A69925&node-id=1597%3A71549&viewport=366%2C48%2C0.12&scaling=scale-down&starting-point-node-id=1597%3A71549](https://www.figma.com/proto/NdNx5jV2R2IDeMflnWPMD7/Issuing-NFT?page-id=1597%3A69925&node-id=1597%3A71549&viewport=366%2C48%2C0.12&scaling=scale-down&starting-point-node-id=1597%3A71549)

# Recordings
👤 Tester 1️⃣ 
https://tldv.io/app/meetings/61c19f6503e0c000138d3081
👤 Tester 2️⃣
https://tldv.io/app/meetings/61c1e5f203e0c000138d3892

# User sessions
👤 Tester 1️⃣
- Notices NFT Preview and says:
> So this is your NFT preview - sure, fair enough!
- Reads very carefully all steps before proceeding to choose anything inside the form
- Checks minimum bid and understands how it works
- Understand that the time duration feature would set up the time of the auction
- Checks the fee at the action bar (the familiar place from the previous testing)
- Scans the summary
- While scanning moves through payment split quickly - saying: ok.
- When asked about payment split:
> The concept is clear, I'm issuing NFT so I'm a creator - I'm selling it so, this is how much I will get after the purchase
- He misunderstood the concept of split - he later states that the wording payment split is misleading.
- When asked about the area on the left:
> Preview of my NFT - I believe how its gonna look on the market

👤 Tester 2️⃣
- Didn't understand a mission at first (didn't read it carefully though - almost skipped it)
- When asked how would he issue his video as an NFT:
> I would upload the video and convert it to MP4 and then issue it on one of the marketplaces.
- He didn't understand at first that this is exactly the feature we are testing now (I think that this is my mistake - I should introduce him to the topic more before testing the prototype with him)
- He clicks the next step after switching Issue as NFT
- Reads the description that shows up 
> So you can upload your video as NFT that's incredible. What a cool feature that would be!
- When approaching the time duration concept, he thinks that time duration is portioning your video to pieces and selling it as NFT (like NBA top shots, etc.) So he misunderstood the concept. 
> when I upload the videos on the pioneer we call it time. So maybe that's why I misunderstood it.
- When asked about the area on the left:
> I figured that this would be the preview of my NFT
- When asked about the payment split concept:
> It's just where the royalties go, if it's gonna be sold for 100k it's where this money would go. I wouldn't think it's a secondary sale. Maybe wording issue - the word payment right of the bat - if it would say royalty it would be more clear.


# When asked about how was it for them to issue an NFT this way:
👤 Tester 1️⃣
It's simpler now. Previously it wasn't clear with all those options in boxes.

👤 Tester 2️⃣ 
EZ. Incredibly easy. That's the easiest - normally they are never as fluid as that - you cannot upload it and list it at the same time. That was fluid you can control.

# Summary
- Users went through the prototype successfully and completed the given task.
- Improvement could be noticed in the overall understanding of the flow so getting rid of additional screens and additional logic (open, English auction, etc) was the right choice.
- NFT preview feature was understandable for the users and they noticed that it changes along with our changes in the form so it successfully serves its purpose
- The new extended summary (with information about fee and royalties) is easy to scan - users had no problems with it 
- **Both users** had problems with the Payment split section - they didn't understand that this relates to the royalties concept - they thought its a split of revenue from their initial sell
- Tester 2 had a problem with the "time duration" concept - it was unclear that it directly relates to the auction itself - he thought it allows him to cut his video in pieces and sell one video as multiple NFT's (on the other hand tester 1 understood it correctly)

# Recommendations
- Payment split - is not the right wording for this particular concept. As its misleading for the users as it is I recommend changing title to "Royalties split" and adding small helper text to explain this concept more clearly.
- Time duration - was not clear enough - recommended change in title to "listing duration."
- Current version of the flow seems to be easy and understandable for the users - so except small wording issues I would recommend to start preparing those screens for development

**Comment by KubaMikolajczyk** [comment]
🟢  Only 2 users participated in this round of tests.
The next test round will be done on 04/Jan/22 to gather more valuable insights.

**Comment by KubaMikolajczyk** [comment]
# Overview:
After the last round of moderated tests, we arrived at 3 final improvements in Wizard flow:
1️⃣  Explainer field was changed for a live "your NFT preview" due to low discoverability and overall usefulness for the users
2️⃣  Fee that is shown in the bottom action bar need to be also presented in the final summary for ease of scanning
3️⃣  Second screen of the form where the user chooses listing type (open, English, fixed etc) need to be dropped - there is no need to introduce those abstract concept to users as those confused all testers - instead we will show all possible options in one form under "put on marketplace"

If you want to know more about previous test click here -> https://github.com/Joystream/atlas/issues/1694 

### Amount of participants: 8

# Questions
### Have you already uploaded any video to Joystream studio?
👤 Tester 1️⃣  ❌
👤 Tester 2️⃣  ✅
👤 Tester 3️⃣  ❌
👤 Tester 4️⃣  ❌
👤 Tester 5️⃣  ❌
👤 Tester 6️⃣  ❌
👤 Tester 7️⃣  ✅
👤 Tester 8️⃣  ❌

### Have you bought or issued any NFT?
👤 Tester 1️⃣  ✅ 
👤 Tester 2️⃣  ✅
👤 Tester 3️⃣  ✅
👤 Tester 4️⃣  ❌
👤 Tester 5️⃣  ❌
👤 Tester 6️⃣  ❌
👤 Tester 7️⃣  ✅
👤 Tester 8️⃣  ❌

### What marketplaces have you already used?
👤 Tester 1️⃣  Opensea
👤 Tester 2️⃣  Opensea, Singular, Pankake swap, check out new marketplaces
👤 Tester 3️⃣  Singular
👤 Tester 4️⃣  Doesn't know what it is.
👤 Tester 5️⃣  Didn't use it.
👤 Tester 6️⃣  Doesn't know the concept.
👤 Tester 7️⃣  Opensea, Singular
👤 Tester 8️⃣  ‎Luard, marketplace na lunie

### Have you participated in the previous Issuing NFT test?
👤 Tester 1️⃣  ✅
👤 Tester 2️⃣  ❌
👤 Tester 3️⃣  ❌
👤 Tester 4️⃣  ❌
👤 Tester 5️⃣  ❌
👤 Tester 6️⃣  ❌
👤 Tester 7️⃣  ✅
👤 Tester 8️⃣  ❌

# Task
"You are publishing a video to Joystream studio, all video assets and information have been prefilled for you. Now you want to sell your video as an NFT and you don’t want to get bids lower than 100 000 tJOYS"

#### Prototype link:
[https://www.figma.com/proto/NdNx5jV2R2IDeMflnWPMD7/Issuing-NFT?page-id=1597%3A69925&node-id=1597%3A71549&viewport=366%2C48%2C0.12&scaling=scale-down&starting-point-node-id=1597%3A71549](https://www.figma.com/proto/NdNx5jV2R2IDeMflnWPMD7/Issuing-NFT?page-id=1597%3A69925&node-id=1597%3A71549&viewport=366%2C48%2C0.12&scaling=scale-down&starting-point-node-id=1597%3A71549)

# Recordings
All recordings will be located in this folder 👇


# User sessions
👤 Tester 1️⃣
- Notices NFT Preview and says:
> So this is your NFT preview - sure, fair enough!
- Reads very carefully all steps before proceeding to choose anything inside the form
- Checks minimum bid and understands how it works
- Understand that the time duration feature would set up the time of the auction
- Checks the fee at the action bar (the familiar place from the previous testing)
- Scans the summary
- While scanning moves through payment split quickly - saying: ok.
- When asked about payment split:
> The concept is clear, I'm issuing NFT so I'm a creator - I'm selling it so, this is how much I will get after the purchase
- He misunderstood the concept of split - he later states that the wording payment split is misleading.
- When asked about the area on the left:
> Preview of my NFT - I believe how its gonna look on the market

👤 Tester 2️⃣
- Didn't understand a mission at first (didn't read it carefully though - almost skipped it)
- When asked how would he issue his video as an NFT:
> I would upload the video and convert it to MP4 and then issue it on one of the marketplaces.
- He didn't understand at first that this is exactly the feature we are testing now (I think that this is my mistake - I should introduce him to the topic more before testing the prototype with him)
- He clicks the next step after switching Issue as NFT
- Reads the description that shows up 
> So you can upload your video as NFT that's incredible. What a cool feature that would be!
- When approaching the time duration concept, he thinks that time duration is portioning your video to pieces and selling it as NFT (like NBA top shots, etc.) So he misunderstood the concept. 
> when I upload the videos on the pioneer we call it time. So maybe that's why I misunderstood it.
- When asked about the area on the left:
> I figured that this would be the preview of my NFT
- When asked about the payment split concept:
> It's just where the royalties go, if it's gonna be sold for 100k it's where this money would go. I wouldn't think it's a secondary sale. Maybe wording issue - the word payment right of the bat - if it would say royalty it would be more clear.

👤 Tester 3️⃣
- understands the mission goal and understands that he has to toggle the switch to list as an NFT
- Reads all of the subtitles, tooltips and banners. Later he mentions
> It was really intuitive due to all of those tooltips.
- Gets confused by payment split section 
- Thinks that this is the share of his primary price instead of it being a share of royalties.
- Understands the concept of the transaction fee
- Didn't understand the concept of creator and owner at first - analysis of the NFT preview helped with understanding later on 
- When asked about the NFT preview area.
> I just thought is just like a thumbnail for my nft so I know what Im listing Showing me information of what Im selling at the moment.

👤 Tester 4️⃣  
- Went through all steps relatively fast
- Noticed the steps counter and comments
> Its nice that I can see on which step I am
- Says that its a really simple process
- He wasn't still quite sure after the test why someone would want to issue video as an NFT
- He noticed that nft preview suggests with its interactivity that you should be able to play video from here
- Has problems with payment split - misunderstands the concept
- This user didn't like the fast that there is only dark mode and wishes to see a light mode switch as well in the future 

👤 Tester 5️⃣  
- Notices preview of NFT and identifies it correctly right away.
- He reads that the form takes 3 steps
- He tries to go back from Choose listing type step and reacts positively to the switch being automatically unchecked, allowing to just upload without issuing
- Correctly identifies the difference between put on the marketplace and not for sale
- Notices that he doesn't know if this will be changeable later or is it a permanent choice.
- Correctly identifies section with listing summary 
- Payment split section creates confusion - He thinks that platform will take 2% of this transaction
-Asked about preview area identifies it correctly
- Notices that hover effects here are not needed

👤 Tester 6️⃣  
- Reads all tooltips carefully
- Understands the difference between different listing types
- Doesn't understand the concept of payment split
- Has the same problem with ;logic here as other users

👤 Tester 7️⃣  
- Reads the banner and subtitles
- Fills the 100 minimum bid
- Defines time duration as something that resembles of cutting his video into pieces - confirms that auction duration would be more clear here as wording
- Correctly identifies the preview area on the left as a preview of his final NFT here


👤 Tester 8️⃣  
- He wonders for a moment should it be public or unlisted
- Understands the concept of time duration
- Has the same problem with understanding the payment split concept


# When asked about how was it for them to issue an NFT this way:
👤 Tester 1️⃣
It's simpler now. Previously it wasn't clear with all those options in boxes.

👤 Tester 2️⃣ 
EZ. Incredibly easy. That's the easiest - normally they are never as fluid as that - you cannot upload it and list it at the same time. That was fluid you can control.

👤 Tester 3️⃣
Really easy I felt this was really intuitive. Even the person who just started using PC and understands English can do this.

👤 Tester 4️⃣  
It was really easy.

👤 Tester 5️⃣  
Cool - I liked it - it was a clear flow. Its good that it is a step by step experience

👤 Tester 6️⃣  
I like that there is so many tooltips where you can learn and that there is information about video being locked upfront.

👤 Tester 7️⃣  
Super EZ. All this is so black and white.

👤 Tester 8️⃣  
Super. I would make decision of making it NFT or not much more prominent. Maybe at the top of the screen.

# Additional questions from participants:
- ‎How many tjoys is 30$.
- What will happen with tJoys when Joystream moves to mainnet?
- Do I retain my copyrights after I sell my video as an NFT?
- Do I acquire copyrights when I buy NFT?
- Can I have unlisted video as NFT?
- How to prevent creator from using the same content again as an NFT?

# Summary
- Users went through the prototype successfully and completed the given task.
- Improvement could be noticed in the overall understanding of the flow so getting rid of additional screens and additional logic (open, English auction, etc) was the right choice.
- NFT preview feature was understandable for the users and they noticed that it changes along with our changes in the form so it successfully serves its purpose
- The new extended summary (with information about fee and royalties) is easy to scan - users had no problems with it 
- All users had problems with the Payment split section - they didn't understand that this relates to the royalties concept - they thought its a split of revenue from their initial sell
- Two testers had a problem with the "time duration" concept - it was unclear that it directly relates to the auction itself - he thought it allows him to cut his video in pieces and sell one video as multiple NFT's (on the other hand tester 1 understood it correctly)

# Recommendations
- Payment split section was confused as a concept in all tests. I recommend getting rid of this section from here, changing its title to royalties split (also confirmed by users to be more understandable) and showing it as part of listing summary if user sets any royalties in auction
- Time duration - was not clear enough - recommended change in title to "Auction duration."
- Add information to choose listing type screen that this can be changed later and the choice is not permanent
- Remove interactivity from NFT Preview component - leave just hovers on owner and creator avatars as those still can be used to get more information about this NFT, but thumbnail hover suggest that we could play a video here and that's why we should remove this.
- Current version of the flow seems to be easy and understandable for the users so I would recommend to finish this phase of testing/ repair found potential issues and prepare final screens for development

**Comment by dmtrjsg** [comment]
@KubaMikolajczyk  great work, excellent analysis ⭐ ! 

Totally support all your recommendations here and wanted to add a few more points from my observations of the flow:

**My five cents:**

1. Studio screen split between the sections to be adjusted from 50/50 to 1/1.61803 (to match the NFT flow)
2. Funnel steps indication moves from below the video preview to above the form - so perhaps worthwhile investigating retaining it in either place for both flows - tbc
3. "Show more options" - should that be expanded by default when Issue as NFT set to true? Do we even need to hide them behind this additional chevron? (how frequent do we expect these to be updated?)
4. The original Wizard had pretty nice looking explainers with graphics on each auction option - would it make sense to fit some of this to the "preview" area of NFT for easier visual explanation of the options?
5. Royalties split - totally agreed, plus the tooltips should do the job!

________

**Answers to the questions:**
1️⃣  The up to date Exchange Rate is published here > https://testnet.joystream.org/#/tokenomics. Right at the moment its 26.96 USD/1MJOY. So $30 will get them 1,112,918 TJoys.
2️⃣ Upon transition to Mainnet, TJoys earned on TestNet can be cashed in. We use Bitcoin Cash (BCH) as a payment rail on our testnets. You can cashout tJOY by making a transfer to the dedicated burn address: 5D5PhZQNJzcJXVBxwJxZcsutjKPqUPydrvpu6HeiBfMaeKQu Before doing this, you should set the memo of the key from which the transfer is made to your BCH address. At the moment of the transfer, the BCH to USD exchange rate is locked. More info is here: https://www.joystream.org/token/
3️⃣ pending with @dmtrjsg 
4️⃣ pending with @dmtrjsg 
5️⃣ pending with @dmtrjsg 
6️⃣ pending with @dmtrjsg

**Comment by KubaMikolajczyk** [comment]
@dmtrjsg thank you 🙇‍♂️ 

#### As for your comments:
1. I remember when you first initiated this idea. I made a version of a screen following this ratio as a test, but after that, we had a group discussion (on one of our Wednesday meetings I believe) where the final group decision was to stay consistent with the rest of our screens and choose the proposed 1:1 ratio for this flow. Besides, I believe that all designers agreed on that and are using the 1:1 ratio in mentioned NFT flow.
2. Stepper component discoverability and readability is really high according to the tests (many users read it, notice it, and are happy about the fact it is showing them the progress in form). What is more, those are two different flows - providing video assets & listing your NFT - and the use of the stepper component is different (in the first flow, you can delete the asset, wheres in the second you are just informed about place in flow) - so I would recommend to stick with the current placement for both of those.
3. Just to clarify "Show more setting" button is a separate thing to issue as NFT toggle - it allows the user to expand the form and get more optional settings for the video itself. ( So in this case it wouldn't make sense for us to expand it when issue as NFT is set to true). 
TLDR: This "show more" button is used in both flows - providing video assets & listing your NFT - and it's performing a crucial role in decluttering the main part of the form so the user can focus on the most important fields in both forms - it also follows hick's law: The time it takes to make a decision increases with the number and complexity of choices. So in this case we are breaking complex form into smaller elements in order to decrease the cognitive load. I'm also looking forward to seeing analytics for those fields that we decided to put under show more settings and see if there is any room for improvement.
4. I would say no to that - 2 iterations ago when we still had all left field dedicated to visual explainers, during the tests, those were almost not noticeable at all, and right now as we developed a pattern of live NFT preview which responds to user actions in the form on the right - I wouldn't add more visual distractions to this screen which, as far as we know, do not add much to the general experience. Gladly we are covered in the field of explaining things as pattern of tooltips being attached to those more complicated elements in our flow was found really nice & helpful by testers.

Thank you for providing answers to those additional questions 🙇‍♂️🏆
